src/components/hint/index.tsx

```diff
@@ -232,11 +279,15 @@ class Hint extends BaseComponent {
 
   getHintPosition() {
     const {position} = this.props;
-    const hintPositionStyle = {left: -this.targetLayout.x, alignItems: 'center'};
+    const hintPositionStyle: HintPositionStyle = {alignItems: 'center'};
+
+    if (this.targetLayout?.x) {
+      hintPositionStyle.left = -this.targetLayout.x;
+    }
 
-    if (position === HINT_POSITIONS.TOP) {
+    if (position === HintPositions.TOP) {
       hintPositionStyle.bottom = 0;
-    } else {
+    } else if (this.targetLayout?.height) {
```

Comment:
Are you sure this `if` will not break anything? Cause you can set it `undefined`...